src/policy/fees.h

```diff
@@ -10,6 +10,7 @@
 #include <uint256.h>
 #include <random.h>
 #include <sync.h>
+#include <txmempool.h>
```

Comment:
You can remove the txmempool forward declarations below if including the header.